lib/selinux/selinux_linux.go

```diff
@@ -37,10 +37,10 @@ const (
 )
 
 // copied from github.com/opencontainers/selinux/go-selinux/selinux-linux.go
-func readConfig(target string) string {
+func readConfig(target string) (string, error) {
 	in, err := os.Open(selinuxConfig)
 	if err != nil {
-		return ""
+		return "", trace.Wrap(err)
```

Comment:
nit: because the caller wraps this with a fairly generic "failed to find SELinux type" message, it would probably improve user experience if this included a message long the lines of `"failed to open %q for reading", selinuxConfig`.

That said, os.Open might already include something along these lines in its error message... can't remember.

Reply:
`os.Open` will list the path that it failed with, so we should be fine here